<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlabGUI</class>
 <widget class="QMainWindow" name="SlabGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>69</y>
      <width>20</width>
      <height>6000</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>60</y>
      <width>6000</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="uphorizontalLayout">
     <item>
      <widget class="QComboBox" name="modelPhaseComboBox">
       <item>
        <property name="text">
         <string>Line Diagram</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Properties</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Loads</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mesh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Results</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="deleteObjectButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="editObjectButton">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="calculateButton">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="viewButton">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>241</width>
      <height>203</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="leftverticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="addSurfaceButton">
       <property name="text">
        <string>Add Surface Geometry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addSupportConditionsButton">
       <property name="text">
        <string>Add Support Conditions</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
